server/api/db/sqldb/db.py

```diff
@@ -2503,6 +2503,9 @@ def create_project(self, session: Session, project: mlrun.common.schemas.Project
             session, project_record.name, raise_on_not_found=False
         )
         if not project_summary:
+            logger.debug(
+                "Creating project summary in DB", project_name=project.metadata.name
+            )
```

Comment:
if you delete the table upon project deletion - why do we need to perform get or create rather than create? old copy might have stale records. I think better of creating it because the summary should not be there

Reply:
IIRC I added it I think because of error handling since we want to get a conflict on the project if it already exists. I agree we should probably change this. @moranbental can you please verify against `tests.api.api.test_projects.test_create_project_failure_already_exists` that you get the expected result?

Reply:
@alonmr The test failed because the `get_identifier_string` method was missing